TeamCode/src/main/java/suitbots/GyroTest.java

```diff
@@ -9,12 +9,15 @@
 
 @TeleOp (name = "Gyro Test")
 
-public class GyroTest extends OpMode {
+public abstract class GyroTest extends OpMode {
     private Robot robot;
     @Override
-    public void init() {
+    public void init_loop() {
```

Comment:
`init_loop()` can't block, either. So if the gyro is calibrated, you want to say calibrated, if it's not you just say it's not, and always you call `telemetry.update()` so you can be sure that your data makes it to the phone.